rc/tools/go/gopls.kak

```diff
@@ -55,22 +55,22 @@ define-command -hidden -params 1 gopls-cmd %{
     gopls-prepare
     evaluate-commands %sh{
         dir=${kak_opt_gopls_tmp_dir}
-        gopls $1 -w ${kak_buffile} 2> "${dir}/stderr"
+        gopls "$1" -w "${kak_buffile}" 2> "${dir}/stderr"
         if [ $? -ne 0 ]; then
             # show error messages in *debug* buffer
-            printf %s\\n "echo -debug %file{ '${dir}/stderr' }"
+            printf %s\\n "echo -debug %file{${dir}/stderr}"
         fi
-        rm -r "${dir}"
     }
     edit!
+    evaluate-commands %sh{ rm -r "${kak_opt_gopls_tmp_dir}" }
```

Comment:
If we don't care about the commands printed by the shell we can use the `nop` command (`-f` added for completeness):

```suggestion
    nop %sh{ rm -rf "${kak_opt_gopls_tmp_dir}" }
```

Reply:
I would not use `-f` because we expect the directory to exist, so printing an error if it doesn't seems right. It's not very important though.

Reply:
I use it because it's only a temporary directory, so its being there doesn't impact the functionality of the tool, and we can afford to ignore unexpected/unlikely errors.